src/renderers/webgl/WebGLRenderLists.js

```diff
@@ -137,13 +137,32 @@ function WebGLRenderList() {
 
 	}
 
+	// remove any dangling references to make sure no memory is leaked
+	// when the count of objects in a scene is reduced
+	function cleanup() {
+
+		for ( var i = this.renderedItemCount; i < renderItems.length; i ++ ) {
```

Comment:
Shouldn't this be...

```js
for ( var i = renderItemsIndex; i < renderItems.length; i ++ ) {
```

Reply:
It should. Silly me. Should be fixed now.

Reply:
@OndrejSpanel May I ask where `renderedItemCount` originally came from? This property does not exist in the entire file. Is it from a forked version of `three.js`?

Reply:
I think it was my lack of attention combined with autocomplete. I think I wrongly started typing `this.renderItem` and (as I have tested to confirm now) this is what autocomplete offers me in this situation (member of `DomRepeatElement`).